src/data/enums/MoveResult.ts

```diff
@@ -0,0 +1,5 @@
+export enum MoveResult {
+  None,
+  ExitToTown,
+  EnterDungeon,
+}
```

Comment:
We might want to add an `EnterCombat` value as it's one of the outcomes, this would also allow us to easily move the combat check out of the `movePlayerToCell` function to be more in line with the new flow, though this would require some additional logic.

The `None` value should be renamed to `Default` or similar, as `MoveResult.None` implies that no movement took place.